### PR TITLE
PRC-156 : Build the PREPROD environment

### PR DESCRIPTION
**Pull request by prabash-moj**
PRC-156 : Build the PREPROD environment

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: "applicationinsights.dev.json"
     API_BASE_URL_HMPPS_AUTH: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
     API_BASE_URL_PRISONER_SEARCH: "https://prisoner-search-preprod.prison.service.justice.gov.uk"
-    FEATURE_EVENTS_SNS_ENABLED: false
+    FEATURE_EVENTS_SNS_ENABLED: true
```
Comment (timharrison-moj) [inline helm_deploy/values-preprod.yaml:15]: we can leave this `false` for now - we're not emitting any sync events at present.

File: .circleci/config.yml [hunk 2]
```diff
@@ -113,7 +113,7 @@ workflows:
     jobs:
       - hmpps/gradle_owasp_dependency_check:
           cache_key: "v2_0"
-          jdk_tag: "21.0"
+          jdk_tag: << pipeline.parameters.java-version >>
           slack_channel: << pipeline.parameters.alerts-slack-channel >>
```
Comment (timharrison-moj) [inline .circleci/config.yml:117]: This change is erroring the build. Could be spaces inside the <<   >>, or missing pipeline parameter for java-version.

Comment (prabash-moj) [reply .circleci/config.yml:120]: thx, updated, build is fixed now 👍

File: .circleci/config.yml [hunk 3]
```diff
@@ -65,42 +65,42 @@ workflows:
...
+      - hmpps/deploy_env:
+          name: deploy_preprod
+          env: "preprod"
+          jira_update: true
+          jira_env_type: staging
+          pipeline_id: <<pipeline.id>>
+          pipeline_number: <<pipeline.number>>
+          context:
+            - hmpps-common-vars
+            - hmpps-contacts-api-preprod
```
[the remark this answers is not in the document]
Comment (prabash-moj) [reply .circleci/config.yml]: created bootstrap PR with context": "hmpps-contacts-preprod", and context": "hmpps-contacts-prod",

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**